src/modules/dashboard/components/ColonyHome/ColonyMembers/MembersSubsection.tsx

```diff
@@ -120,27 +120,28 @@ const MembersSubsection = ({
             </div>
           }
         >
-          <NavLink to={membersPageRoute}>
-            <Heading
-              appearance={{ size: 'normal', weight: 'bold' }}
-              text={MSG.title}
-              textValues={{
+          <ClickableHeading linkTo={membersPageRoute}>
+            <FormattedMessage
+              {...MSG.title}
+              values={{
                 count: members?.length,
                 hasCounter,
                 isContributorsSubsection,
               }}
             />
-          </NavLink>
+          </ClickableHeading>
         </Tooltip>
         {!isContributorsSubsection && (
-          <InviteLinkButton
-            colonyName={colonyName}
-            buttonAppearance={{ theme: 'blueWithBackground' }}
-          />
+          <div className={styles.inviteButton}>
+            <InviteLinkButton
+              colonyName={colonyName}
+              buttonAppearance={{ theme: 'blueWithBackground' }}
+            />
+          </div>
         )}
       </div>
     ),
-    [members, membersPageRoute, isContributorsSubsection, colonyName],
+    [isContributorsSubsection, membersPageRoute, members],
```

Comment:
github shows me that you have a missing dependency `colonyName`

Reply:
Good shout, thanks!